components/ScheduleHome.vue

```diff
@@ -21,6 +32,29 @@
         </div>
       </div>
     </div>
+    <div v-if="false" class="old-schedule">
+      <div v-for="(day, index) in showsByDate" :key="day.id" class="mb-2 scheduleblock">
+        <div class="flex flex-row justify-between dayname">
+          <h4>{{ dayNames[day[0].day-1] }}</h4>
+          <span>{{ format(new Date(index === 0 ? todayDate :
+            tommorrow.setDate(tommorrow.getDate() + 1)) , 'MMMM dd.') }}</span>
+        </div>
+        <div class="dayshows">
+          <div v-for="show in day" :key="show.id" :class="showAirCheck(index, show.name) ? 'dayblock onair' : 'dayblock'">
+            <div class="show-basic-infos">
+              {{ removeSeconds(show.start) }}-{{ removeSeconds(show.end) }} -
+              <nuxt-link :to="'/shows/' + show.archive_lahmastore_base_url">
+                {{ show.name }}
+              </nuxt-link>
+              <div class="show-image" :style="{ backgroundImage: `url(${show.cover_image_url})` }" />
```

Comment:
Wow! Awesome! I was just thinking about adding the show cover image for each show in the schedule

Reply:
not the nicest solution, but it works, and I was really amazed to see that vue is loading these images on-demand and therefore reducing the bandwidth strain

Reply:
we can discuss this further on Saturday how this hover image show is the nicest, and maybe also to add some other meta info to this popup

Reply:
This was one of my favourites during the demo. Great idea & implementation! 

Vue is cool too they really played their part in this based on what you're writing! 

I would like to have more meta info in there for sure!